### PR TITLE
Data-balloons

### PR DESCRIPTION
**Pull request by RukayaDavids**
Added the data balloons for the new elements added in v10

### CODE REVIEW

File: language/en [hunk 1]
```diff
@@ -201,5 +201,17 @@
 			"remove_title": "Remove Photo",
 			"confirm_remove": "Are you sure you want to remove this Photo?"
 		}
-	}
+	},
+	"draw_tip": "Draw on Canvas",
+	"add_text": "add new text to canvas",
```
Comment (alcinovanrooyen) [inline language/en]: @RukayaDavids Why with some you start capital letter and others you do not ?

Comment (RukayaDavids) [reply language/en]: @alcinovanrooyen I added the value's from the html in our demo page. Please let me know how I should edit it.

Comment (alcinovanrooyen) [reply language/en]: @RukayaDavids which demo page ?

Comment (RukayaDavids) [reply language/en]: Any one but you can look at this one https://wp.demo.pitchprint.io/product/blue-business-card/. When you right click the elements it shows the words like that.

Comment (RukayaDavids) [reply language/en]: @alcinovanrooyen please look at this one https://wp.demo.pitchprint.io/product/blue-business-card/.

Comment (alcinovanrooyen) [reply language/en]: @RukayaDavids I will close this pull request but not merge it, as the file has been changed by Isaac on server, will download the file and manually add your updates from this pull request to it. Thanks for this.

**Pull request closed without merging**